src/router.js

```diff
@@ -12,37 +12,41 @@ export class Router {
   }
 
   get(...args) {
-    this.register(...['GET', ...args]);
+    this.request('GET', ...args);
   }
 
   post(...args) {
-    this.register(...['POST', ...args]);
+    this.request('POST', ...args);
   }
 
   put(...args) {
-    this.register(...['PUT', ...args]);
+    this.request('PUT', ...args);
   }
 
   delete(...args) {
-    this.register(...['DELETE', ...args]);
+    this.request('DELETE', ...args);
+  }
+
+  request(method, ...args) {
```

Comment:
Not sure about this name, `request` looks more like doing the actual request to me... What do you think?